readme.md

```diff
@@ -2867,6 +2867,22 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
 | ------: | -------: | -----: |
 |   2 - 5 |      10+ | 45-60m |
 
+### [Terra Mystica](http://www.terra-mystica-spiel.de/en/)
```

Comment:
The link is to a German language page, the contributing guide specifies a link to either Wikipedia or Board Game Geek as a preference, although an English language source from the manufacturer with an https protocol would be allowed.